resource-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/resource/ScenarioPersistenceTest.java

```diff
@@ -87,7 +87,7 @@
     @Autowired
     private ResourceClientWrapper client;
 
-    private static final ConnectionDetail DETAIL = new SimpleConnectionDetail().uri("http://localhost:8082");
+    private static final ConnectionDetail DETAIL = new SimpleConnectionDetail().serviceName("http://localhost:8082");

```

Comment:
Can this be changed so it isn't a URI value since the value isn't actually used at all
```suggestion
    private static final ConnectionDetail DETAIL = new SimpleConnectionDetail().serviceName("test-data-service");

```